src/database/db.ts

```diff
@@ -0,0 +1,16 @@
+import { Sequelize } from "sequelize";
+import { log } from "../utils/logger";
+import { config } from "./config";
+
+const sequelize = new Sequelize(
+  config.database,
+  config.username,
+  config.password,
+  {
+    dialect: "postgres",
+    host: config.host,
+    logging: sql => log.debug("%s", sql)
```

Comment:
```suggestion
    logging: log.debug
```

Reply:
If I change the code as suggested, then I get the following error when executing the migrations:
```
TypeError: self.log is not a function
    at Object.DerivedLogger.(anonymous function) [as logging] (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/winston/lib/winston/create-logger.js:95:19)
    at Sequelize.log (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/sequelize/lib/sequelize.js:1182:15)
    at Query._logQuery (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/sequelize/lib/dialects/abstract/query.js:338:22)
    at Query.run (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/sequelize/lib/dialects/postgres/query.js:62:27)
    at runHooks.then.then (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/sequelize/lib/sequelize.js:641:29)
    at tryCatcher (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:517:31)
    at Promise._settlePromise (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:574:18)
    at Promise._settlePromise0 (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:619:10)
    at Promise._settlePromises (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/promise.js:699:18)
    at _drainQueueStep (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/async.js:138:12)
    at _drainQueue (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/async.js:131:9)
    at Async._drainQueues (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/async.js:147:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/alexgpeppe/IdeaProjects/IO/io-onboarding-pa/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
```